flexxamples/howtos/flask_server.py

```diff
@@ -0,0 +1,77 @@
+from flask import Flask, current_app, url_for
+from flask_sockets import Sockets
```

Comment:
Can you add a docstring explaining what this example does? I think the idea is to embed a Flexx app in a Flask application, but some more details about how this works would be nice.

Reply:
Done.